src/test/CharterManager-unit.t.sol

```diff
@@ -303,17 +306,6 @@ contract CharterManagerTest is TestBase {
         assertEq(gem.balanceOf(address(b)), 200e6);
     }
 
-    function testFail_join_unauthorized() public {
-        (Usr a,) = init_user();
-        a.join(address(a) ,address(a) ,100 * 1e6);
-    }
-
-    function testFail_exit_unauthorized() public {
-        (Usr a,) = init_user();
-        a.join(10 * 1e6);
-        a.exit(address(a), address(a) ,10 * 1e6);
-    }
-
```

Comment:
I guess these were removed as they are move tests related to the `join` than the `charter`.

Reply:
Actually these tests were ported from crop join but I recently realized they don't make much sense since they pass `address(a)` (address of Usr contract) as the gemJoin address.
So I removed them.
I don't think charter manager needs `unauthorized` tests for join/exit, since both rely on msg.sender and don't have an explicit from/src parameter.